pages/[slug].tsx

```diff
@@ -1,18 +1,22 @@
-import type { Category, ExtendedStory } from '@prezly/sdk/dist/types';
+import type { Category, ExtendedStory, Newsroom } from '@prezly/sdk/dist/types';
 import { GetServerSideProps, NextPage } from 'next';
 import { getPrezlyApi } from '@/utils/prezly';
 import Story from '@/modules/Story';
 import Layout from '@/components/Layout';
+import { NewsroomContext } from '@/utils/prezly/context';
 
 type Props = {
     story: ExtendedStory;
     categories: Category[];
+    newsroom: Newsroom;
```

Comment:
We should probably create a common `BasePageProps` type, just not to specify it each time and have less imports. And then just:
```
type Props = BasePageProps & {
    story: ExtendedStory;
};
```